utilities/update_pom_version.sh

```diff
@@ -11,8 +11,7 @@
 # Get the previous maven project version.
 CURRENT_VERSION=$(mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.version | grep -Ev '(^\[|\w+:)')
 # Get list of directories for which pom.xml must be updated
-module_folders=($(find . -maxdepth 1 -name 'gcloud-java*' -type d) .)
-
+module_folders=($(find . -maxdepth 2 \( -regex "./gcloud-java[-[a-z]+]+" -o -regex "./gcloud-java-contrib/gcloud-java[-[a-z]+]+" \) -type d) . ./gcloud-java)
```

Comment:
-maxdepth controls how far find looks for potential matching files (rather than being a "test" condition on whether or not to include a file).  For that reason I believe it has to be kept as 2.